src/web/client/dal/fileSystemProvider.ts

```diff
@@ -242,11 +244,29 @@ export class PortalsFS implements vscode.FileSystemProvider {
     }
 
     async rename(): Promise<void> {
-        throw new Error("Method not implemented.");
+        oneDSLoggerWrapper.getLogger().traceError(
+            telemetryEventNames.WEB_EXTENSION_RENAME_NOT_SUPPORTED,
+            WEB_EXTENSION_METHOD_NOT_IMPLEMENTED,
+            { name: telemetryEventNames.WEB_EXTENSION_RENAME_NOT_SUPPORTED, message: WEB_EXTENSION_METHOD_NOT_IMPLEMENTED} as Error,
+        );
+        WebExtensionContext.telemetry.sendErrorTelemetry(
+            telemetryEventNames.WEB_EXTENSION_RENAME_NOT_SUPPORTED,
+            this.rename.name
+        );
+        throw new Error(WEB_EXTENSION_METHOD_NOT_IMPLEMENTED);
     }
 
     async delete(): Promise<void> {
-        throw new Error("Method not implemented.");
+        oneDSLoggerWrapper.getLogger().traceError(
+            telemetryEventNames.WEB_EXTENSION_DELETE_NOT_SUPPORTED,
+            WEB_EXTENSION_METHOD_NOT_IMPLEMENTED,
+            { name: telemetryEventNames.WEB_EXTENSION_DELETE_NOT_SUPPORTED, message: WEB_EXTENSION_METHOD_NOT_IMPLEMENTED} as Error
```

Comment:
same as above

Reply:
can you refer to the above for this too.